RRTPlanner:
  robot_radius: 2
  goal_tolerance: 12
  path_tolerance: 10
  path_checkpoint_resolution: 1 # do not change
  obstacle_inflation_radius: 1
  time_step: 0.3
  num_step: 3
  linear_velocity_max: 1.0
  angular_velocity_max: 1.0
  linear_velocity_min: 0.0
  angular_velocity_min: 0.0
  motion_primitive_resolution: 3 # odd numbers work best
  max_iterations: 100
  controller_frequency: 10.0
  local_costmap_border: true
  forward_bias: 6
  velocity_topic: "robot_0/cmd_vel"
